### PR TITLE
PR for Rename sexpr-wasm binaries #100

### PR DESCRIPTION
**Pull request by citisolo**
Hi , I have renamed the binaries  to the suggested names and created some symlinks that keep the testsuite (sexpr-wasm -> wast2wasm and wasm-wast -> wasm2wast) in-tact

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
-[![Build Status](https://travis-ci.org/WebAssembly/sexpr-wasm-prototype.svg?branch=master)](https://travis-ci.org/WebAssembly/sexpr-wasm-prototype)
+[![Build Status](https://travis-ci.org/WebAssembly/wast2wasm-prototype.svg?branch=master)](https://travis-ci.org/WebAssembly/wast2wasm-prototype)
 
-# sexpr-wasm-prototype
```
Comment (sbc100) [inline README.md]: You probably want to revert the changes to `sexpr-wasm-prototype`, which is the current repo name.

File: CMakeLists.txt [hunk 2]
```diff
@@ -223,15 +223,23 @@ add_library(libwasm STATIC
...
 
-# sexpr-wasm
-add_executable(sexpr-wasm src/sexpr-wasm.c)
-add_dependencies(everything sexpr-wasm)
-target_link_libraries(sexpr-wasm libwasm)
+# wast2wasm
+add_executable(wast2wasm src/wast2wasm.c)
+add_dependencies(everything wast2wasm)
+target_link_libraries(wast2wasm libwasm)
+add_custom_command( TARGET wast2wasm
```
Comment (binji) [inline CMakeLists.txt]: This will break on Windows, probably want to just do this conditionally.

File: test/help/sexpr-wasm.txt [hunk 3]
```diff
@@ -1,25 +1,25 @@
 ;;; EXE: %(sexpr-wasm)s
```
Comment (binji) [inline test/help/sexpr-wasm.txt]: Probably want to rename these too, though that can be done later, I suppose.

**Comment by binji** [comment]
Oh, and have you joined the WebAssembly community group?

File: test/help/sexpr-wasm.txt (hunk 3 above)
Comment (citisolo) [inline test/help/sexpr-wasm.txt]: I will renamed those and the tests failed . with the output:
**** FAILED ****************************************************************** 
- help/sexpr-wasm.txt
    
- help/wasm-wast.txt

 I can't quite tell why

Comment (binji) [inline test/help/sexpr-wasm.txt]: Probably because they need to be updated in the python test runner too. It's ok, this change looks good to land. Thanks!
